collect_app/src/androidTest/java/org/odk/collect/android/support/matchers/DrawableMatcher.java

```diff
@@ -26,4 +27,18 @@ public boolean matchesSafely(ImageView imageView) {
             }
         };
     }
+
+    public static Matcher<View> withBackgroundColor(int color) {
+        return new BoundedMatcher<View, View>(View.class) {
+            @Override
+            public void describeTo(Description description) {
+                description.appendText("View background color to be " + color);
+            }
+
+            @Override
+            public boolean matchesSafely(View view) {
+                return color == ((GradientDrawable) view.getBackground()).getColor().getDefaultColor();
```

Comment:
I was going to say that we didn't need to test color here (and should do it at a lower level) but you've found a really simple way to do it! Nice.